app/src/main/java/org/oppia/android/app/topic/practice/TopicPracticeFragmentPresenter.kt

```diff
@@ -69,7 +70,9 @@ class TopicPracticeFragmentPresenter @Inject constructor(
       this.viewModel = viewModel
       lifecycleOwner = fragment
     }
-    return binding.root
+    return binding.root.apply {
+      setBackgroundResource(R.color.topic_activity_layout_background_color)
+    }
```

Comment:
Is this actually needed? I think we can set the background colors using just the xml files instead.

Reply:
Fixed that